### PR TITLE
clarify permission requirements

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#1031 
Source:  I checked the required permission as described [here](https://docs.microsoft.com/powershell/exchange/exchange-server/find-exchange-cmdlet-permissions?view=exchange-ps)

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor edit

File: microsoft-365/security/office-365-security/set-up-atp-safe-attachments-policies.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ People regularly send, receive, and share attachments, such as documents, presen
 
 - Make sure that your organization has [Office 365 Advanced Threat Protection](office-365-atp.md).
 
-- Make sure that you have the necessary permissions. To define (or edit) ATP policies, you must be assigned an appropriate role. Some examples are described in the following table:
+- Make sure that you have the necessary permissions. To define (or edit) ATP policies, you must be assigned to either Exchange Online Organization Management role (Office 365 Global Administrator is assigned to this role by default) or both Exchange Online Hygiene Management and Security Administrator roles. For more details, see the following table:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-atp-safe-attachments-policies.md]:
```suggestion
- Make sure that you have the necessary permissions. To define (or edit) ATP policies, you must be assigned either an Exchange Online Organization Management role (Office 365 Global Administrator is assigned to this role by default) or both Exchange Online Hygiene Management and Security Administrator roles. For more details, see the following table:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #1031

Thank you.

cc: @kenwith

**Pull request merged**